herddb-core/src/main/java/herddb/server/ServerSideConnectionPeer.java

```diff
@@ -103,8 +103,7 @@
     /**
      * Open scanners. The ID is generated by the client
      */
-    private final ConcurrentMap<Long, ServerSideScannerPeer> scanners = new ConcurrentHashMap<>();
-    private final ConcurrentMap<RawString, Set<Long>> openTransactions = new ConcurrentHashMap<>();
```

Comment:
this was the bad mechanism that was binding a transaction to the originating socket (ServerSideConnectionPeer)